src/Core/Snippets/Snippets.cs

```diff
@@ -149,6 +149,14 @@ public Snippet Compile(string code)
 
             try
             {
+                // strip any @EntryPoint() attributes from the snippet
+                var entryPointPattern = @"@\s*EntryPoint\s*\(\s*\)";
+                if (Regex.IsMatch(code, entryPointPattern))
+                {
+                    code = Regex.Replace(code, entryPointPattern, string.Empty);
+                    logger.LogWarning("@EntryPoint()", "@EntryPoint() attributes are ignored when compiling Q# code in this workspace.");
+                }
```

Comment:
The only way I can see to do this is to replace via regex prior to compilation, since the compiler may throw an error. This isn't ideal, since the regex doesn't know whether what it's removing is truly an EntryPoint attribute. If there is a better suggestion for how to do this, please let me know.